File: soalNo04.py
array_input = input("Masukkan angka pemisah dengan koma (,): ")
arr = [int(x) for x in array_input.split(",")]

largest_number = arr[0]
for i in range(1, len(arr)):
    if arr[i] > largest_number:
        largest_number = arr[i]

print(f"Angka terbesar dari array {arr} adalah: {largest_number}")

File: soalNo04.js
// Buatlah program yang menerima input sebuah `array of integers`,
// dan menampilkan angka `terbesar` dari array tersebut

// run dengan node js

// script inputan pada node js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Masukan array angka dipisah dengan koma (,): ", (arrInput) => {
  const arr = arrInput.split(",").map(Number);
  let angkaTerbesar = arr[0];
  for (let i = 1; i < arr.length; i++) {
    if (arr[i] > angkaTerbesar) {
      angkaTerbesar = arr[i];
    }
  }
  console.log(
    `Pada array [${arr}]. Angka terbesarnya adalah: ${angkaTerbesar}`
  );

  rl.close();
});
